tests/e2e/tests/admin/guided-tour.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { test, expect } from '@playwright/test';
+import { sharedSetup } from '../../utils/setup';
+import { STRAPI_GUIDED_TOUR_CONFIG, setGuidedTourLocalStorage } from '../../utils/global-setup';
+import { clickAndWait } from '../../utils/shared';
+
+test.describe('Guided tour', () => {
+  test.beforeEach(async ({ page }) => {
+    await setGuidedTourLocalStorage(page, { ...STRAPI_GUIDED_TOUR_CONFIG, enabled: true });
+
+    // Now proceed with the normal setup (login, etc.)
+    await sharedSetup('guided-tour', page, {
+      login: true,
+      resetFiles: true,
+      importData: 'with-admin.tar',
+    });
+  });
+
+  test('should be greeted with the Guided Tour Overview', async ({ page }) => {
+    await expect(page.getByRole('heading', { name: 'Discover your application!' })).toBeVisible();
+    await expect(page.getByRole('listitem', { name: 'Create your schema' })).toBeVisible();
+    await expect(page.getByRole('listitem', { name: 'Create and publish content' })).toBeVisible();
+    await expect(
+      page.getByRole('listitem', { name: 'Create and copy an API token' })
+    ).toBeVisible();
+    await expect(
+      page.getByRole('listitem', { name: 'Deploy your application to Strapi Cloud' })
+    ).toBeVisible();
+  });
```

Comment:
maybe in another iteration and with another dataset and schema you can check if the Start in the CTB is enabled and the other two are disabled, you can add a comment for it